### ISSUE TITLE
ext/File-DosGlob: status in core-p7 branch

### ISSUE THREAD

**Issue by jkeenan** [atoomic/perl#63]
In this ticket we will track our core-p7 progress on tests in directory `ext/File-DosGlob`.

As 2020-07-09 10:20 EDT, we are getting this output from `make test` for the files in this directory.

```
../ext/File-DosGlob/t/DosGlob.t .................................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 21/21 subtests 
Bareword found where operator expected at t/find.t line 23, near "import File::Find"
	(Do you need to predeclare import?)
syntax error at t/find.t line 23, near "import File::Find"
BEGIN not safe after errors--compilation aborted at t/find.t line 24.

```
These results were generated on an unthreaded build on linux.  The results might differ significantly if run on a threaded build.  There have been subsequent commits to this branch which may have impacted these results.  YMMV.

**Comment by jkeenan** [comment]
ext/File-DosGlob/t/*.t: Updated data at commit e69542876a in p7 branch:
```
$ cat File-DosGlob 
Bareword found where operator expected at t/DosGlob.t line 146, near "import XS::APItest"
	(Do you need to predeclare import?)
String found where operator expected at t/DosGlob.t line 146, near "XS::APItest "sv_count""
	(Do you need to predeclare XS::APItest?)
syntax error at t/DosGlob.t line 146, near "import XS::APItest "
Execution of t/DosGlob.t aborted due to compilation errors.
# Looks like your test exited with 255 before it could output anything.
../ext/File-DosGlob/t/DosGlob.t .................................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 21/21 subtests 
```

**Comment by jkeenan** [comment]
Fixed in 8487407d2f.  Closing.
